awx/api/views/__init__.py

```diff
@@ -3154,9 +3154,10 @@ def get(self, request, *args, **kwargs):
     def post(self, request, *args, **kwargs):
         obj = self.get_object()
         if obj.is_sliced_job:
-            if not obj.job_template_id:
+            jt = obj.job_template
```

Comment:
Will this throw some sort of `DoesNotExist` exception; was there a reason we were checking `job_template_id` before to avoid a join?

Reply:
I use the `_id` fields because it does not do a query. Django default behavior is that a Model object contains FK ids in `self.__dict__`, but not the fields of that related object.

But `obj.job_template` _can_ throw a DNE error, only if the JT was deleted in the time window between fetching the workflow job object and the `getattr`, but this possibility exists virtually everywhere in the app.